CONTRIBUTING.md

```diff
@@ -47,6 +62,26 @@ For more explained information about plugin override check [Development Override
 - Commit and push your changes to your branch then submit a pull request against the `master` branch.
 - A repo maintainer will review the your pull request, and may either request additional changes or merge the pull request.
 
+#### PR Title Format
+We use [*Conventional Commits*](https://www.conventionalcommits.org/):
+- `fix: description of the PR`: a commit of the type fix patches a bug in your codebase (this correlates with PATCH in Semantic Versioning).
+- `chore: description of the PR`: the commit includes a technical or preventative maintenance task that is necessary for managing the product or the repository, but it is not tied to any specific feature or user story (this correlates with PATCH in Semantic Versioning).
```

Comment:
knitpit: I would updare the ticket of this PR to be `chore:` so that the label is removed that is not added to the enhancement section of the changelog. This is a feature for us but not for customers

Reply:
yeah good point, adjusting.